lib/rest-adapter.js

```diff
@@ -369,22 +369,41 @@ RestAdapter.errorHandler = function(options) {
         res.statusCode = err.statusCode || err.status || 500;
       }
 
-      debug('Error in %s %s: %s', req.method, req.url, err.stack);
-      var data = {
-        name: err.name,
-        status: res.statusCode,
-        message: err.message || 'An unknown error occurred',
-      };
-
-      for (var prop in err) {
-        data[prop] = err[prop];
+      if (Array.isArray(err)) {
+        var details = err.map(function(it) {
+          var data = generateResponseError(it);
+          delete data.statusCode;
+          return data;
+        });
+
+        err = new Error('Failed with multiple errors, see `details` for more information.');
+        err.details = details;
       }
 
-      data.stack = err.stack;
-      if (process.env.NODE_ENV === 'production' || options.disableStackTrace) {
-        delete data.stack;
+      debug('Error in %s %s: %s', req.method, req.url, err.stack);
```

Comment:
@bajtos 
I didn't get confused, I just forgot to delete the line outside of `generateResponseError`. I think that the debug log should remain in `generateResponseError` as is. It will display the individual errors from the connector as well as the location in which they were aggregated.